lib/src/runner/browser/browser_manager.dart

```diff
@@ -225,7 +225,7 @@ class BrowserManager {
           sink.close();
         }));
 
-    return await _pool.withResource(() async {
+    return await _pool.withResource/*<Future<RunnerSuite>>*/(() async {
```

Comment:
I've sort of given up on bug notes for analysis issues like this TBH. It's too much work to track down and verify which one attaches to which bug. At some point when inference is more solid I'll just go through looking for explicit generic annotations and seeing if I can remove them.